package.json

```diff
@@ -6,20 +6,58 @@
     "dev": "next dev --turbopack",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "dev:server": "ts-node --project tsconfig.server.json server/server.ts"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "^2.8.2",
+    "@types/axios": "^0.14.4",
+    "@types/cookie-parser": "^1.4.9",
```

Comment:
Check your dependencies carefully - it sometimes happens that a dependency is insalled to `dependencies` instead of `devDependencies`, as e.g. types here - it might negatively affect the bundle size in some cases.